src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -91,3 +92,14 @@ def _complex_params(command_group, engine):
         with self.argument_context(scope) as c:
             c.argument('server_name', options_list=['--server-name', '-s'])
             c.argument('configuration_name', id_part='child_name_1', options_list=['--name', '-n'])
+
+    for scope in ['postgres server vnet-rule', 'postgres server vnet-rule']:
```

Comment:
These two scopes are identical, so you don't need  a list. I suspect you meant to have two scopes, one for create and one for update. However, just exposing these at this scope (vnet-rule) will work just fine, as it will be ignored on commands that don't have these parameters.

Reply:
I meant one for mysql .. I ll update.